components/templates/Function/index.js

```diff
@@ -3,6 +3,10 @@
 var React = require('react');
 
 var TemplateFn = React.createClass({
+  propTypes: {
+    template: React.PropTypes.string,
```

Comment:
And my editor was complaining about them missing since our use of react eslint plugin